bin/node/cli/benches/block_production.rs

```diff
@@ -19,7 +19,7 @@
 use criterion::{criterion_group, criterion_main, BatchSize, Criterion, Throughput};
 
 use node_cli::service::{create_extrinsic, FullClient};
-use node_runtime::{constants::currency::*, BalancesCall};
+use node_kitchensink_runtime::{constants::currency::*, BalancesCall};
```

Comment:
I wonder if the word `node` is reasonable here anymore -- should this not be just `kitchensink-runtime`, placed under `bin/kitchensink/runtime`? 

IDK, but in my head the word 'node' is just noise there.